newapi/ooniapi/citizenlab.py

```diff
@@ -535,10 +486,32 @@ def url_submission_update_url():
               type: string
             comment:
               type: string
-            new_entry:
-              type: array
             old_entry:
-              type: array
+              type: object
+              properties:
+                category_code:
+                  type: string
+                url:
+                  type: string
+                date_added:
+                  type: string
+                user:
+                  type: string
+                notes:
+                  type: string
+            new_entry:
```

Comment:
I think this also implies that the type for `updated_entry` in the response should also be changed to `object`.

Reply:
Updated. Is it useful or should we drop it?